### PR TITLE
elixir #4390: setup normalizer tests

### PR DESCRIPTION
**Pull request by wobh**
A couple of tests confirming that the decomposition misalignment is gone and that normalization works.

### CODE REVIEW

File: lib/elixir/test/elixir/string/normalizer_test.exs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+Code.require_file "../test_helper.exs", __DIR__
+
+defmodule String.NormalizerTest do
+  use ExUnit.Case, async: true
+
+  # String.normalize
+
+  test "String.normalize 015C;LATIN CAPITAL LETTER S WITH CIRCUMFLEX" do
```
Comment (lexmag) [inline lib/elixir/test/elixir/string/normalizer_test.exs]: Maybe better to group all the `String.normalize` tests under one `test` like:
```elixir
test "String.normalize/2" do
  # 015C;LATIN CAPITAL LETTER S WITH CIRCUMFLEX
  assert String.normalize("\u015C", :nfd) == "\u0053\u0302"
  assert String.normalize("\u0053\u0302", :nfc) == "\u015C"
  # 015D;LATIN SMALL LETTER S WITH CIRCUMFLEX
  assert String.normalize("\u015D", :nfd) == "\u0073\u0302"
  assert String.normalize("\u0073\u0302", :nfc) == "\u015D"
  # ...
end
```
?

Comment (lexmag) [inline lib/elixir/test/elixir/string/normalizer_test.exs]: It seem easier to read.

**Comment by josevalim** [comment]
@wobh I have included those tests already when I fixed those issues. :) I was thinking we could include the whole normalization test file rather, similar to the script I posted on another thread. That should cover all bases. Thank you! :heart:

**Comment by wobh** [comment]
Agreed. I only submitted that since I hadn't been sure about how to do the comprehensive test and this would be better than nothing. Since you did figure out the comprehensive test this would be superfluous.

**Pull request closed without merging**